src/lib/index.js

```diff
@@ -451,8 +451,12 @@ lib.addStyleRule = function(selector, styleString) {
     else lib.warn('addStyleRule failed');
 };
 
-lib.getTranslate = function(element) {
+// TODO (alexcjohnson): the following translate/scale functions should
+// get moved to components/drawing rather than the generic lib.
```

Comment:
:+1: for moving `getTranslate` and `setTranslate` out of `Lib`. Maybe putting them into `Drawing` would make the most sense?